go/kbfs/libkbfs/kbfs_ops_test.go

```diff
@@ -3539,6 +3539,7 @@ func TestKBFSOpsMaliciousMDServerRange(t *testing.T) {
 
 	// Create mallory's fake TLF using the same TLF ID as alice's.
 	config2 := ConfigAsUser(config1, "mallory")
+	defer func() { _ = config2.Shutdown(ctx) }()
```

Comment:
Why the extra closure? `config2` isn't re-assigned after this.

Reply:
Oh did you mean to require no error like the one below?

Reply:
No, this one fails if we require no error due to the intentional errors of the test.  But if the `err` is not assigned to `_` (which requires the closure) then lint will fail.